content/tags.md

```diff
@@ -123,7 +123,7 @@ cucumber --tags "@smoke and @fast"
 <p></p>
 
 {{% tip "Filtering by line" %}}
-Another way to run a subset of scenarios is to use the `file.feature:line` pattern or the `--scenario` option as described in [Running Features](/configuration/#running-features).
+Another way to run a subset of scenarios is to use the `file.feature:line` pattern or the `--scenario` option as described in [Running Features](/running).
```

Comment:
Change link "[Running Features]" to "Running Cucumber"?

Reply:
Done